manifests/supervisorcluster/1.21/cns-csi.yaml

```diff
@@ -178,6 +179,10 @@ spec:
               value: "6443"
             - name: VSPHERE_CLOUD_OPERATOR_SERVICE_PORT
               value: "29000"
+            - name: VSPHERE_CLOUD_OPERATOR_SERVICE_NAME
```

Comment:
Add a comment here describing the relationship between the flag and the env variables.